sprint/faq.md

```diff
@@ -0,0 +1,17 @@
+# Frequently Asked Questions
+
+## Do I need to be a PyMC user to participate in the open source working session?
+
+**Response:**  It is not necessary to be a PyMC user to contribute to this open source project. There are a number of issues where we could use your contribution. Examples:  
+- Adding timestamps to videos from our PyMCon conference
+- Updating docstrings in documentation
+- Update notebooks to v4.  This video, [Contributing to PyMC Documentation](https://youtu.be/fzpmLWQNj4A) provides a background and examples. 
+
+
+## Is there a requirement to have attended #1 of the working session?
+
+**Response:** The Pre-event office hours are an opportunity for incoming contributors to ask questions for the upcoming working sessions.  It is *not required* to attend a specific session as a prerequisite for other sessions.  Ideally, it is beneficial if folks can attend as many sessions as they can, to build up their open source skills and familiarity with the PyMC project. 
+
+If you are unable to attend the Pre-event office hours, you can ask your questions in one of these ways:  
+- Open an issue on the Data Umbrella & PyMC [Open Source Event Website's GitHub repo](https://github.com/pymc-devs/pymc-data-umbrella/issues)
```

Comment:
Would it be too much friction to have the questions on discourse? We can add a category and/or tag for them. I am not sure the distinction between questions about the working sessions but not about the contributions themselves or about pymc will be clear to participants. And the 2nd ones should clearly go on discourse.